### ISSUE TITLE
keybase pgp gen hangs on 

### ISSUE THREAD

**Issue by duaneking** [keybase/keybase-issues#2942]

**Comment by duaneking** [comment]
It goes without saying but I'm saying it: This kills functionality since I cant generate a key otherwise.

I'm dfk on keybase.io if that helps with debugging, its just not proofed yet because I can't even get that far.

**Comment by duaneking** [comment]
Yes I'm aware of the fact this act is insecure.. but you support it and per I wanted to see if it would work. https://github.com/keybase/keybase-issues/issues/160

**Comment by zapu** [comment]
Hello, thanks for the report. Can you send a log and post id here? The command is: `keybase log send`
